reactor-test/src/main/java/reactor/test/publisher/ColdTestPublisher.java

```diff
@@ -456,7 +457,7 @@ public ColdTestPublisher<T> assertNoRequestOverflow() {
 	}
 
 	@Override
-	public ColdTestPublisher<T> next(@Nullable T t) {
+	public ColdTestPublisher<T> next(@Nullable T t, Consumer<T> onUndelivered) {
```

Comment:
Is there no need to use the `onUndelivered` value like in `DefaultTestPublisher`?

Reply:
I rolled back this API since it is a breaking one after all. In the case of `ColdTestPublisher`, all the values are cached in the internal store, so they can be consumed slightly latter